aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, Grid, Stack, Text } from 'grommet';
+import PropTypes from 'prop-types';
+
+// values chosen to best visually represent the margin area in diagram
+const largeColumns = ['13px', 'auto', '13px'];
+const smallColumns = ['8px', 'auto', '8px'];
+
+const MarginText = ({ label }) => (
+  <Stack anchor="bottom">
+    <Text alignSelf="center">{label}</Text>
+    {/* values chosen to best visually represent horizontal line in diagram. */}
+    <Box width="12px" height="1px" background="black" />
+  </Stack>
+);
+
+const GridViewPort = ({ columns, label, marginLabel, width }) => (
+  <Box gap="small" width={width}>
+    <Box direction="row" justify="between">
+      <MarginText label={marginLabel} />
+      <MarginText label={marginLabel} />
+    </Box>
+    <Grid
+      columns={columns}
+      // values chosen to best visually represent the margin area in diagram
+      rows={['20px', 'small', '20px']}
+      areas={[
+        ['left-margin', 'top-gap', 'right-margin'],
+        ['left-margin', 'viewpoint', 'right-margin'],
+        ['left-margin', 'bottom-gap', 'right-margin'],
+      ]}
+      border={{ style: 'dashed' }}
+      // border radius needed to match figma had to use in style
+      style={{ borderRadius: '4px' }}
+    >
+      <Box gridArea="top-gap" background="background-front" />
+      <Box
+        id="leftMargin"
+        gridArea="left-margin"
+        background="validation-critical"
+      />
+      <Box
+        id="rightMargin"
+        gridArea="right-margin"
+        background="validation-critical"
+      />
+      <Box
+        gridArea="viewpoint"
+        align="center"
+        justify="center"
+        round="xsmall"
+        background="status-unknown"
+        border={{ style: 'dashed' }}
+      >
+        <Text color="text-strong" weight="bold">
+          {label}
+        </Text>
+      </Box>
+      <Box gridArea="bottom-gap" background="background-front" />
+    </Grid>
+  </Box>
+);
+
+export const PageMarginAnatomy = () => (
+  <Box gap="large" direction="row">
```

Comment:
```suggestion
  <Box gap="large" direction="row-responsive">
```
should this go to single column on mobile?

Reply:
I wasn't sure about mobile view but I like the single column view